doc/README.md

```diff
@@ -237,10 +237,45 @@ them. This is useful to combine profiles from multiple processes of a
 distributed job. The profiles may be from different programs but must be
 compatible (for example, CPU profiles cannot be combined with heap profiles).
 
-pprof can subtract a profile from another in order to compare them. For that,
-use the **-base= _profile_** option, where *profile* is the filename or URL for the
-profile to be subtracted. This may result on some report entries having negative
-values.
+
+## Comparing profiles
+
+pprof can subtract one profile from another, provided the profiles are of
+compatible types. pprof has two options which can be used to specify the
+filename or URL for a profile to be subtracted from the source profile:
+
+* **-diff_base= _profile_:** useful for comparing any two profiles of compatible
```

Comment:
Both -diff_base and -base require the profiles to be compatible so the focus on the compatibility of profiles in this paragraph is somewhat unexpected to me? I'd mention that in the paragraph above.

Reply:
done.